package.json

```diff
@@ -33,6 +33,25 @@
 			{
 				"command": "godot-tool.run_project",
 				"title": "Godot Tools: Run workspace as Godot project"
+			},
+			{
+				"command": "godot-tool.ShowDoc",
```

Comment:
We use snake_case in this project to make contributors form godot project comfotable.